app/src/main/res/drawable/ic_aboutapp.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24"
+    android:viewportWidth="24">
+
+    <path
+        android:pathData="M0 0h24v24H0z"/>
+    <path
+        android:fillColor="@color/facebook_primary"
+        android:pathData="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1
+15h-2v-6h2v6zm0-8h-2V7h2v2z"/>
+</vector>
```

Comment:
Again use attrs file instead of colors. keep this as color primary or color primary dark whichever suits according to you.
We are going to use blue as primary color..no need to specify it in this pr as #72 will include that